Sources/XCHammer/XcodeTarget.swift

```diff
@@ -416,6 +416,8 @@ public class XcodeTarget: Hashable, Equatable {
             return xcTargetName + ".xctest"
             case .AppExtension, .XPCService, .Watch1App, .Watch2App, .Watch1Extension, .Watch2Extension, .TVAppExtension:
             return xcTargetName + ".appex"
+            case .StaticLibrary:
```

Comment:
Thanks for fixing this 👍 . As a followup, we should probably make sure that every single value in this enum is handled here correctly ( removing `default` ). cc @rahul-malik